### ISSUE TITLE
accuracy  problems,

### ISSUE THREAD

**Issue by 1771203081** [open-mmlab/mmskeleton#320]
I followed the instructions completely and used the data provided in Google drive, but the accuracy of ST-GCN was extremely low in NTU-RGB-D-xsub

**Comment by happysheep224** [comment]
you mean you use the pre-train model or train a new model by using dataset  NTU-RGB-D-xsub

**Comment by 1771203081** [comment]
> you mean you use the pre-train model or train a new model by using dataset NTU-RGB-D-xsub

 train a new model

**Comment by happysheep224** [comment]
Yeah, I think I met the same problem with you. I use the original model to train, in theory, its accuracy should as same as the pre-train model. But the accuracy is extreme low, top-1 1%, top-5 8%.

**Comment by zjr666** [comment]
So how did you solve it

**Comment by huangjun12** [comment]
The same problem

**Comment by bkleck** [comment]
Hi is there any reason for this? I made use of the pre-trained model and got pretty low accuracies of 1-8% too

**Comment by Infin1te2021** [comment]
Same issue I met. I loaded the [pretrained weight](https://github.com/open-mmlab/mmskeleton/blob/master/checkpoints/st_gcn.ntu-xsub-300b57d4.pth) for 60 classes X-sub testing and all predictions were trending towards Action 22 (cheer up).
No data augmentation, no shuffle
```
{'loss': 17859166012.479164, 'accuracy': 0.01710438527324559, 'top_5_acc': 0.0838842724570874}
```

![Image](https://github.com/user-attachments/assets/e7fa95b7-7dae-4fc0-92c1-e57916c0d1e0)

---
(Update) Remember to pad 0 to present the missing body and skeleton joint value instead of other values such as -1e9. After the correction, the results I tested were acceptable.
 (The result of padding the minor_body with the major_bdoy, missing frame with 0)
```
{'loss': 1.553059420707394, 'accuracy': 0.7651482986595499, 'top_5_acc': 0.9498392673015102}
```
![Image](https://github.com/user-attachments/assets/81aabaaf-f523-4000-9dd9-3b61b4e4de5a)

**Now, the performance matches the description on paper**
 (The result of padding the minor_body with the 0, missing frame with 0) 
```
{'loss': 0.8190340051918819, 'accuracy': 0.815430339055013, 'top_5_acc': 0.9683993449384364}
```

![Image](https://github.com/user-attachments/assets/cbc169f2-db72-4076-937a-34c49a6ba35d)
